etc/gerbil.el

```diff
@@ -334,15 +336,24 @@
                        "type-of"
                        "spawn" "spawn*" "spawn/name" "spawn/group"
                        ;; sugar
-                       "try" "finally" "catch"
-                       "while" "until" "using"
-                       "hash" "hash-eq" "hash-eqv" "let-hash"
+                       "try" "finally" "catch" "with-destroy"
+                       "while" "until" "using" "defmethod/alias"
+                       "with-methods" "with-class-methods" "with-class-method"
+                       "hash" "hash-eq" "hash-eqv" "let-hash" "assert!" "awhen"
+                       "chain" "is"
                        ;; coroutines
                        "continue" "yield" "coroutine"
                        ;; iterators
                        "for" "for*" "for/collect" "for/fold"
                        ;; actor messaging
                        "<-" "<<" "->"
+                       ;; test
+                       "run-tests!" "test-suite" "test-case"
+                       "test-report-summary!" "run-test-suite!" "test-result"
```

Comment:
and these two are just procedures used in scripts orchestrating tests, I don't think they merit special font locking.